packages/create-llama/helpers/index.ts

```diff
@@ -54,6 +54,10 @@ const createEnvLocalFile = async (
 
   if (opts?.llamaCloudKey) {
     content += `LLAMA_CLOUD_API_KEY=${opts?.llamaCloudKey}\n`;
+  } else {
+    content += `# Please obtain the Llama Cloud API key from https://cloud.llamaindex.ai/api-key 
```

Comment:
this would copy this content all the time - even if using llamacloud is not configured, right?

Reply:
Thanks, it's a good catch.